components/elm/cime_config/config_component.xml

```diff
@@ -68,10 +68,13 @@
       <value compset="PDAY.*_ELM"            >1850-2100_rcp4.5_transient</value> 
       <!-- CMIP6 DECK compsets and related BGC compsets -->
       <value compset="1850(?:SOI)?_EAM%CMIP6_ELM">1850_CMIP6_control</value>
+      <value compset="1850(?:SOI)?_EAM%CMIP6_ELM.*_BGC%LNDATM">1850_CMIP6bgc_control</value>
+      <value compset="1850(?:SOI)?_EAM%CMIP6_ELM.*_BGC%BCRC">1850_CMIP6bgc_control</value>
       <value compset="1850S_EAM%CMIP6_ELM">1850_SCMIP6_control</value>
       <value compset="1850(?:SOI)?_EAM%CMIP6_ELM*_BGC%BCRC">1850_CMIP6bgc_control</value>
       <value compset="1850S_EAM%CMIP6_ELM">1850_SCMIP6_control</value>
       <value compset="20TR.*_EAM%.*CMIP6.*_ELM">20thC_CMIP6_transient</value>
+      <value compset="20TR_EAM%CMIP6_ELM.*_BGC%BLNDATM">20thC_CMIP6bgc_transient</value> 
```

Comment:
should this be "BGC%BLNDATM" or just "BGC%LNDATM"?

Reply:
just "BGC%LNDATM